app/views/tag/_tags.html.erb

```diff
@@ -0,0 +1,18 @@
+<p>
+  <span id="tags">
```

Comment:
@gaeun Since id should be unique keep these outer span tags in the `tagging` partial surrounding the render tags.